### PR TITLE
[WIP] API documentation

### PR DESCRIPTION
**Pull request by epissari**

### CODE REVIEW

File: src/controllers/demographicDataController.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ class DemographicDataController {
             .then(result => {
                 if (result.length !== 1) {
                     res.status(404).send('User not found');
+<<<<<<< HEAD
```
Comment (fguitton) [inline src/controllers/demographicDataController.js]: This is NOT OK !

**Comment by fguitton** [comment]
@epissari A much greater care should be given to the following.
If the git engine complains there is a reason. **Don't simply ignore and commit anyway !**

https://github.com/dsi-icl/optimise-core/blob/797ff90638290f778b3280e74a56368f6cbdcd00/src/controllers/demographicDataController.js#L39-L43

https://github.com/dsi-icl/optimise-core/blob/797ff90638290f778b3280e74a56368f6cbdcd00/src/controllers/demographicDataController.js#L249-L258

File: src/controllers/demographicDataController.js (hunk 1 above)
Comment (epissari) [reply src/controllers/demographicDataController.js]: Corrected in commit 00f2f4cf5ca4b1a37db9ed6607c1e421555e62a6 !!

**Comment by epissari** [comment]
@fguitton ec0b401dd16c36dbc52834bb3ed2cead64ab4181 is not the final commit- I will be following what I did for the TreatmentInterruption for the next commits.

**Comment by fguitton** [comment]
@epissari Is that now finished ?

**Comment by epissari** [comment]
@fguitton Yes

**Comment by epissari** [comment]
@fguitton I need to do a final check for the response object structure.

**Pull request merged**